site-scrapers/CVS.js

```diff
@@ -0,0 +1,43 @@
+const sites = require("../data/sites.json");
+
+const siteName = "CVS";
+const site = sites[siteName];
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log(`${siteName} starting.`);
+    const webData = await ScrapeWebsiteData(browser);
+    console.log(`${siteName} done.`);
+    return webData.map((responseLocation) => {
+        let hasAvailability = parseInt(responseLocation.totalAvailable)
+            ? true
+            : false;
+        let availability = responseLocation.totalAvailable;
+        return {
+            name: `${siteName} (${responseLocation.city})`,
+            hasAvailability,
+            availability,
+            signUpLink: site.website,
+            ...responseLocation,
+        };
+    });
+};
+
+async function ScrapeWebsiteData(browser) {
+    let rawData = {};
+    const page = await browser.newPage();
+    await page.goto(site.website, { waitUntil: "domcontentloaded" });
+    await page.waitForSelector("a[data-modal='vaccineinfo-MA']");
+    const [searchResponse, ...rest] = await Promise.all([
+        Promise.race([
+            page.waitForResponse(
+                "https://www.cvs.com/immunizations/covid-19-vaccine.vaccine-status.ma.json?vaccineinfo"
```

Comment:
If we're going to start at the toplevel CVS website, we should not hardcode the Mass. JSON page.
(Or we could go directly to the Mass. JSON page).

Reply:
Good call. I moved it to the sites obj.